static/js/base.js

```diff
@@ -100,3 +105,16 @@ function stop() {
     update({X:144,Y:144,dir:0});
     clearInterval(timer);
 };
+
+function clearConsole(){
+  document.getElementById("consoleLog").innerText = ""
+}
+clearConsole()
+
+function log(text) {
+  if(!text.includes('pyodide.py')){
+    if(!text.includes('<eval>')){
+  logged = document.getElementById("consoleLog").innerText += text + "\n";
+    }
+  }
```

Comment:
Rather than using nested if statements, it would be nice to create a list of strings to be filtered, loop through them, and return without altering the console if any of the strings are in the message.